src/main/kotlin/no/nav/familie/ef/sak/service/AvstemmingService.kt

```diff
@@ -38,8 +38,8 @@ class AvstemmingService(private val oppdragClient: OppdragClient,
 
     fun konsistensavstemOppdrag(stønadstype: Stønadstype) {
         val oppdragIdListe = tilkjentYtelseService.finnLøpendeUtbetalninger(datoForAvstemming = LocalDate.now(), stønadstype = stønadstype)
-        val konsistensavstemmingRequest = KonsistensavstemmingRequest(fagsystem = stønadstype.tilKlassifisering(),
-                                                                      oppdragIdListe = oppdragIdListe,
+        val konsistensavstemmingRequest = KonsistensavstemmingRequestV2(fagsystem = stønadstype.tilKlassifisering(),
+                                                                      periodeIdn = oppdragIdListe,
```

Comment:
Småpirk, `periodeIder` er kanskje norsk for `periodeIdn` ?

Reply:
endret til perioder/perioderForBehandlinger